tests/RobotFramework/tests/tedge_to_te_converter/convert_tedge_topics_to_te_topics.robot

```diff
@@ -18,70 +18,70 @@ Convert main device measurement topic
     Should Be Equal    ${messages_tedge}  ${messages_te}    
 
 Convert main device empty measurement topic
-    Execute Command    tedge mqtt pub tedge/measurements '{"temperature":25}'
-    ${messages_tedge}=    Should Have MQTT Messages    tedge/measurements    minimum=1    maximum=1
-    ${messages_te}=    Should Have MQTT Messages    te/device/main///m/    minimum=1    maximum=1
+    Execute Command    tedge mqtt pub tedge/measurements '{"temperature":20}'
+    ${messages_tedge}=    Should Have MQTT Messages    tedge/measurements    minimum=1    maximum=1    message_contains=20
+    ${messages_te}=    Should Have MQTT Messages    te/device/main///m/    minimum=1    maximum=1   message_contains=20
     Should Be Equal    ${messages_tedge}  ${messages_te}
-    Should Have MQTT Messages    te/device/main///m/    message_pattern={"temperature":25}
+    Should Have MQTT Messages    te/device/main///m/    message_pattern={"temperature":20}
    
 Convert child device measurement topic
     Execute Command    tedge mqtt pub tedge/measurements/child '{"temperature":25}'
-    ${messages_tedge}=    Should Have MQTT Messages    tedge/measurements/child   minimum=1    maximum=1
-    ${messages_te}=    Should Have MQTT Messages    te/device/child///m/    minimum=1    maximum=1
+    ${messages_tedge}=    Should Have MQTT Messages    tedge/measurements/child   minimum=1    maximum=1    message_contains=25
+    ${messages_te}=    Should Have MQTT Messages    te/device/child///m/    minimum=1    maximum=1    messsage_contains=25
     Should Be Equal    ${messages_tedge}  ${messages_te}
     Should Have MQTT Messages    te/device/child///m/    message_pattern={"temperature":25}
 
 Convert main device event topic
     Execute Command    tedge mqtt pub tedge/events/login_event '{"text":"someone logedin"}'
```

Comment:
Trivial. I feel weird to the word `logedin`. Should have been "logged in", missing one `g`..